modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -523,38 +520,57 @@ public void start() {
     }
 
     private void performRebalanceOnRecovery(long recoveryRevision) {
-        CompletableFuture<Void> pendingAssignmentsRecoveryFuture;
-
-        var prefix = new ByteArray(PENDING_ASSIGNMENTS_PREFIX);
+        var pendingAssignmentsPrefix = new ByteArray(PENDING_ASSIGNMENTS_PREFIX);
```

Comment:
Why did you change performRebalanceOnRecovery? rebalance is about handling pending keys. I expect it to be properly implemented in IGNITE-20344. Did you find bugs there?

Reply:
Probably it should be renamed to `processAssignmentsOnRecovery`. We will need to process both kinds of assignments anyway.

Reply:
Seems that it's not true for now. On node restart we handle stable assignments directly with `startLocalPartitionsAndUpdateClients` without calling `handleAssignmentsOnRecovery`.